test/test_rake_win32.rb

```diff
@@ -3,55 +3,6 @@
 
 class TestRakeWin32 < Rake::TestCase # :nodoc:
 
-  Win32 = Rake::Win32 # :nodoc:
-
-  def test_win32_system_dir_uses_home_if_defined
-    ENV["HOME"] = 'C:\\HP'
-
-    assert_equal "C:/HP/Rake", Win32.win32_system_dir
-  end
-
-  def test_win32_system_dir_uses_homedrive_homepath_when_no_home_defined
-    ENV["HOME"]      = nil
-    ENV["HOMEDRIVE"] = "C:"
-    ENV["HOMEPATH"]  = '\\HP'
-
-    assert_equal "C:/HP/Rake", Win32.win32_system_dir
-  end
-
-  def test_win32_system_dir_uses_appdata_when_no_home_or_home_combo
-    ENV["APPDATA"]   = "C:\\Documents and Settings\\HP\\Application Data"
-    ENV["HOME"]      = nil
-    ENV["HOMEDRIVE"] = nil
-    ENV["HOMEPATH"]  = nil
-
-    assert_equal "C:/Documents and Settings/HP/Application Data/Rake",
-                 Win32.win32_system_dir
-  end
-
-  def test_win32_system_dir_fallback_to_userprofile_otherwise
-    ENV["HOME"]        = nil
-    ENV["HOMEDRIVE"]   = nil
-    ENV["HOMEPATH"]    = nil
-    ENV["APPDATA"]     = nil
-    ENV["USERPROFILE"] = "C:\\Documents and Settings\\HP"
-
-    assert_equal "C:/Documents and Settings/HP/Rake", Win32.win32_system_dir
-  end
-
-  def test_win32_system_dir_nil_of_no_env_vars
-    ENV["APPDATA"]     = nil
-    ENV["HOME"]        = nil
-    ENV["HOMEDRIVE"]   = nil
-    ENV["HOMEPATH"]    = nil
-    ENV["RAKE_SYSTEM"] = nil
-    ENV["USERPROFILE"] = nil
-
-    assert_raises(Rake::Win32::Win32HomeError) do
-      Win32.win32_system_dir
-    end
```

Comment:
As per the above comment, this test is superfluous, as the exception will *never* be thrown IRL.